apps/logsdrilldown/kinds/logsdrilldown.cue

```diff
@@ -1,10 +1,23 @@
 package kinds
 
-logsdrilldownv0alpha1: {
+LogsDrilldownSpecv1alpha1: {
+	defaultFields: [...string] | *[]
```

Comment:
The problem with this schema is that it's mixing user preferences with global preferences, unless you planned on setting those 3 booleans globally.